conf/modules.config

```diff
@@ -700,6 +700,70 @@ process {
         ext.args = '--mitochondria-mode TRUE'
         }
 }
+//
+// ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT
+//
+process {
+    withName: '.*ANALYSE_MT:PREPARE_GENOME_MT:PREPARE_GENOME:BWAMEM2_INDEX' {
+        ext.when = {!params.bwamem2_index && params.aligner == "bwamem2"}
+        publishDir = [
+            path: { "${params.outdir}/references" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+            enabled: params.save_reference,
+            pattern: "*{0123,amb,ann,bwt.2bit.64,bwt.8bit.32,pac}"
+        ]
+    }
+
+
+
+    withName: '.*ANALYSE_MT:PREPARE_GENOME_MT:PREPARE_GENOME:SAMTOOLS_FAIDX' {
+        publishDir = [
+            path: { "${params.outdir}/references" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+            enabled: params.save_reference,
+            pattern: "*fai"
+        ]
+    }
+
+    withName: '.*ANALYSE_MT:PREPARE_GENOME_MT:PREPARE_GENOME:GET_CHROM_SIZES' {
+        publishDir = [
+            enabled: false,
+        ]
+    }
+    withName: '.*ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT:BWAMEM2_MEM_MT' {
+        ext.args = { "-M -K 100000000 -R \"@RG\\tID:${meta.id}\\tSM:${meta.sample}\\tLB:${meta.library}\\tPL:illumina\"" }
+    }
+
+    withName: '.*ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT:GATK4_MERGEBAMALIGNMENT_MT' {
+        ext.args = '--CREATE_INDEX true --MAX_GAPS -1  --SORT_ORDER queryname --INCLUDE_SECONDARY_ALIGNMENTS false --PAIRED_RUN false --VALIDATION_STRINGENCY LENIENT'
+        ext.prefix = { "${meta.id}_merged" }
+    }
+
+    withName: '.*ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT:PICARD_ADDORREPLACEREADGROUPS_MT' {
+        ext.args = '--VALIDATION_STRINGENCY LENIENT'
+    }
+
+    withName: '.*ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT:PICARD_MARKDUPLICATES_MT' {
+        ext.args = '--VALIDATION_STRINGENCY LENIENT --CREATE_INDEX true'
+        ext.prefix = { "${meta.id}_markduplicates" }
+    }
+
+    withName: '.*ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT:SAMTOOLS_SORT_MT' {
+        ext.prefix = { "${meta.id}_sorted" }
+    }
+
+
+    withName: '.*ANALYSE_MT:ALIGN_AND_CALL_MT_SHIFT:GATK4_MUTECT2_MT' {
+        ext.args = '--mitochondria-mode TRUE'
+        }
+    withName: '.*ANALYSE_MT:PICARD_LIFTOVERVCF' {
+        ext.args = ''
+        }
```

Comment:
I guess you can remove this if you are not changing the defaults for that module